### ISSUE TITLE
[WCAG 2.2 Draft Feedback] Success Criterion 3.2.6 Consistent Help (A)

### ISSUE THREAD

**Issue by dshoukry** [w3c/wcag#1892]
"Success Criterion 3.2.6 Consistent Help (Level A): For each web page within a set of web pages that provides one or more of the following ways of finding help, access to at least one form of help is included in the same relative order on each page:
Human contact details;
Human contact mechanism;
Self-help option;
A fully automated contact mechanism.
Access to help mechanisms may be provided directly on the page, or may be provided via a direct link to a different page containing the information."

We appreciate that much of our feedback was accepted here in the first round. Our outstanding request from the last round is a request to reorder the affordances to meet this requirement to prioritize asynchronous help options that can happen at the user's preferred time, rather than human contact that may be restricted to business hours and time zone limitations. Additionally, some companies and customer service agents might not be trained to help people with disabilities, know the etiquette in speaking with PWDs, or be familiar with assistive technology. It may be beneficial to expand the help requirements and provide tips or examples of what might constitute effective self-help options, automated help options, and human help mechanisms.

Please find detailed specifics covered in our [Success Criterion 3.2.6: Consistent Help (fka Findable Help) (A) Google Doc.](https://docs.google.com/document/d/1UXHGHI9Sisv_iQMeAPaQ3DCIoJ8St9yCpd2ndaoUdQ4/edit#)

**Comment by patrickhlauke** [comment]
> Our outstanding request from the last round is a request to reorder the affordances to meet this requirement to prioritize asynchronous help options that can happen at the user's preferred time, rather than human contact that may be restricted to business hours and time zone limitations. Additionally, some companies and customer service agents might not be trained to help people with disabilities, know the etiquette in speaking with PWDs, or be familiar with assistive technology. It may be beneficial to expand the help requirements and provide tips or examples of what might constitute effective self-help options, automated help options, and human help mechanisms.


Note that the list of different types of help aren't actually prioritised, nor is the success criterion actually focused in any way on the details of these help services. It is only normatively concerned that any kind of help that is present on multiple pages is available in the same relative order/place.

See also https://github.com/w3c/wcag/issues/1881

**Comment by rachaelbradley** [comment]
DRAFT Response: The unordered list in the SC text does not imply a priority or recommendation of preference. We have added this clarification to the understanding document at [PR1968 ](https://github.com/w3c/wcag/pull/1968 ).

**Comment by rachaelbradley** [comment]
Note that the response above does not fully close the issue but the other points depend on this first point.

**Comment by alastc** [comment]
Adding the other questions from the doc:

> can we specify that at least one of the following mechanisms to get help should be "active" rather than "included"?
> For example, what happens if a website only offers chat client support as a help mechanism (which would satisfy this recommendation), but there is no human support available at a give time? Or what if support pages are stale and not actively maintained?

The criterion is not making any judgement about when/where support is available, just that it is consistently shown / linked-to on each page in a set. Saying that a mechanism (e.g. phone line) is active is going beyond our scope of web-content guidelines, and is likely un-supportable for many smaller organisations/individuals.


> "For pages for which no human support is available, a self-help option should say that no human support is available. "
> Is the use of should a recommendation or requirement?

That is a recommendation, and since we try to avoid that RFC style language in understanding docs, we should (ahem) update that.


> Could another possible failure be, "inactive help?" If the chatbot or chat client feature is consistently in the same location, but no one is ever available to help, can we consider that a failure?

As above, although it opens a potential hole for people wanting to work around the guidelines, it is beyond our scope to specify offline aspects.

**Comment by alastc** [comment]
Sorry, hadn't updated this issue based on the resolution from: https://www.w3.org/2021/08/17-ag-minutes.html#t05
